sdk/identity/Azure.Identity/CHANGELOG.md

```diff
@@ -58,6 +58,9 @@
 
 ## 1.10.4 (2023-11-13)
 
+### Breaking Changes
+- Added dependency to `WindowsForms` when targeting `netX.0-windows` instead of `netX.0`. An additional installation of .NET Desktop Runtime may be necessary. [#44232](https://github.com/Azure/azure-sdk-for-net/issues/44232)
```

Comment:
```suggestion
- One of Azure.Identity's dependencies, Microsoft.Identity.Client, inadvertently added a dependency to `WindowsForms` when targeting `netX.0-windows` instead of `netX.0` in version 4.56.0. An additional installation of .NET Desktop Runtime may be necessary. Manually adding a reference to the latest Microsoft.Identity.Client will remove the need for the .NET Desktop Runtime. [#44232](https://github.com/Azure/azure-sdk-for-net/issues/44232)
```